### ISSUE TITLE
ValueError: numpy.ndarray size changed

### ISSUE THREAD

**Issue by edogab33** [ddangelov/Top2Vec#193]
I'm running Top2Vec in Databricks and this error appear:

`ValueError: numpy.ndarray size changed, may indicate binary incompatibility. Expected 88 from C header, got 80 from PyObject`

Code:

`from top2vec import Top2Vec

model = Top2Vec(messages_lst, embedding_model='distiluse-base-multilingual-cased')`

Environment:
python3.8.8
numpy==1.20.0 (also 1.21.x doesn't work)
scipy==1.5.2
hdbscan==0.8.27

Seems to be related to hdbscan but doing `!pip install hdbscan --no-cache-dir --no-binary :all: --no-build-isolation` (also with numpy), as suggested in other issues, doesn't solve the problem.

**Comment by jcarletgo** [comment]
Same issue seems to be coming from gensim 
```
  File "/opt/conda/lib/python3.9/site-packages/gensim/matutils.py", line 1104, in <module>
    from gensim._matutils import logsumexp, mean_absolute_difference, dirichlet_expectation
  File ".eggs/Cython-0.29.14-py2.7-linux-x86_64.egg/Cython/Includes/numpy/__init__.pxd", line 242, in init gensim._matutils
ValueError: numpy.ndarray size changed, may indicate binary incompatibility. Expected 88 from C header, got 80 from PyObject
```

**Comment by gxavier38** [comment]
I'm having the same issue running in a Kaggle notebook. I found these relevant issues, but I've tried the suggestions and they didn't work for me. Seems like an old version of top2vec had this issue too.

https://github.com/ddangelov/Top2Vec/issues/125
https://github.com/scikit-learn-contrib/hdbscan/issues/457

**Comment by edogab33** [comment]
Just an update: I was running Top2Vec in Databricks (which was giving me the same problem with BERTopic). Now I migrated everything to Google Colab and it works fine. Probably there is one package which is giving some troubles.

**Comment by gxavier38** [comment]
@edogab33 thanks! Colab also worked for me. That's strange

**Comment by edogab33** [comment]
> This sounds like an environment issue. 

Yes, it is. The problem is that I checked each external library version and they were all correct.
